### PR TITLE
Issue with npm install scripts in test libraries

### PR DESCRIPTION
**Pull request by sarangan12**
In our SDK repository, we have several test libraries created for integration & smoke test scenarios. In these test libraries, we have the `README.md` in which we have installation instructions with `npm install .....` command. But, all these libraries are test libraries and will not be released to npm.  

Now, this causes a security vulnerability. An attacker can create an actual npm library with that name and if someone follows the instruction from our repository and installed that library, it could potentially contain malicious code and could be used for hacking. This scenario has already happened and security team wanted to remove that instruction from our test libraries. This PR is created to remove that instruction from test libraries. 

@deyaaeldeen @joheredi Please review and approve.

@ramya-rao-a FYI.....

### CODE REVIEW

File: src/generators/static/readmeFileGenerator.ts [hunk 1]
```diff
@@ -47,6 +47,8 @@ interface Metadata {
   addCredentials?: boolean;
   /** The link to the identity package in the repository */
   identityPackageURL?: string;
+  /** Indicates if the package is a test/Package package. */
+  isRealPackage?: boolean;
```
Comment (deyaaeldeen) [inline src/generators/static/readmeFileGenerator.ts:51]: I think this flag could use a better name, e.g. `isReleasedPackage`

File: src/utils/autorestOptions.ts [hunk 2]
```diff
@@ -61,6 +63,11 @@ async function getHeadAsBoolean(host: Host): Promise<boolean> {
   return Boolean(headAsBoolean);
 }
 
+async function getIsTestPackage(host: Host): Promise<boolean> {
+  const isTestPackage: boolean = await host.GetValue("is-test-package");
```
Comment (deyaaeldeen) [inline src/utils/autorestOptions.ts:67]: same comment about the name, `is-released-package`

**Review by deyaaeldeen — COMMENTED** [review]
I agree this could be a nasty issue. I left a comment about the name for the flag.

File: src/generators/static/readmeFileGenerator.ts (hunk 1 above)
Comment (sarangan12) [reply src/generators/static/readmeFileGenerator.ts:51]: I am going to change it to `isReleasablePackage`.

File: src/utils/autorestOptions.ts (hunk 2 above)
Comment (sarangan12) [reply src/utils/autorestOptions.ts:67]: Here the is-test-package looks fine

**Pull request merged**